### DISCUSSION TITLE
V3: Login page with real-time validation?

### DISCUSSION THREAD

**Discussion by devhoussam1998** [filamentphp/filament#8040] [Help]
<img width="590" alt="Screenshot 2023-08-24 at 12 57 20" src="https://github.com/filamentphp/filament/assets/121255432/dcff40f5-62c8-4170-8a67-f6593023c591">

**Comment by pboivin** [comment]
https://filamentphp.com/docs/3.x/panels/users#customizing-the-authentication-features
